### PR TITLE
[MMCA-4882] Updating cash account error screens

### PR DESCRIPTION
**Pull request by wg-hmrc**

### CODE REVIEW

**Comment by AdamCoops** [comment]
[error] -- viewmodels/CashAccountV2ViewModel.scala:52:6 
[error] 52 |      new cash_account_balance(emptyH1Component).apply(model = CashAccountViewModel(eori, account))
[error]    |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[error]    |missing argument for parameter h2 of constructor cash_account_balance in class cash_account_balance

**Comment by AdamCoops** [comment]
There are 3 platOpsErrors we can fix as well, they're all deletes in the [conf/application.conf] file. If this has been ear marked for another ticket that's fine to leave it, otherwise I think we just remove them now.

File: test/views/CashAccountExceededThresholdSpec.scala [hunk 1]
```diff
@@ -89,6 +88,17 @@ class CashAccountExceededThresholdSpec extends ViewTestHelper {
 
     linkElement.contains(messages("cf.cash-account.transactions.request.link.pre")) mustBe true
 
+    linkElement.contains(messages("cf.cash-account.transactions.request.link.post")) mustBe true
+
     linkElement.contains(controllers.routes.RequestTransactionsController.onPageLoad().url) mustBe true
   }
+
```
Comment (tyagikumarvineet) [inline test/views/CashAccountExceededThresholdSpec.scala:95]: Remove this extra line break

File: test/views/CashAccountTransactionsNotAvailableSpec.scala [hunk 2]
```diff
@@ -132,6 +136,19 @@ class CashAccountTransactionsNotAvailableSpec extends ViewTestHelper {
   }
 
   private def shouldContainPaymentSection(implicit view: Document): Assertion =
-    view.text().contains(messages("cf.cash-account.detail.transactions-not-available")) mustBe true
+    view.text().contains(messages("cf.cash-account.detail.transactions-not-available")) mustBe false
+
+  private def shouldContainHelpAndSupportSection(implicit view: Document): Assertion =
```
Comment (tyagikumarvineet) [inline test/views/CashAccountTransactionsNotAvailableSpec.scala:141]: Please format from line 142-153 as these are not indented correctly. Also the assertions inside shouldContainHelpAndSupportLink should be moved inside shouldContainHelpAndSupportSection as those are all part of shouldContainHelpAndSupportSection. 

shouldContainHelpAndSupportSection should also be included as part of test named "transactionsTimeout is true"

File: app/views/cash_account_no_transactions_with_balance.scala.html [hunk 3]
```diff
@@ -35,17 +35,13 @@
...
-    @inset(
-        p("cf.cash-account.detail.no-transactions-with-balance.p1").toString(),
-        id = Some("transactions-not-available"),
-        Some(link("cf.cash-account.detail.link",
-            appConfig.cashAccountForCdsDeclarationsUrl,
-            preLinkMessage = Some("cf.cash-account.detail.link.pre"),
-            postLinkMessage = Some("cf.cash-account.detail.link.post")
-        ))
-    )
+    @p("cf.cash-account.detail.no-transactions-with-balance.p1")
```
[the remark this answers is not in the document]
Comment (tyagikumarvineet) [reply app/views/cash_account_no_transactions_with_balance.scala.html]: @wg-hmrc could you please address this comment?

**Review by tyagikumarvineet — APPROVED** [review]

**Pull request merged**